pallav/fashionTech/pallavBanka/parkingAssignment/Parking.java

```diff
@@ -0,0 +1,122 @@
+package fashionTech.pallavBanka.parkingAssignment;
+
+import java.util.*;
+
+class ParkingCharge {
+    Date inTime;
+    Date outTime;
+    private Long ParkingTime(Date inTime, Date outTime) {
+        this.inTime = inTime;
```

Comment:
Why reAssignment?

Reply:
Yeah removed them and merged the methods